python/ml4ir/applications/ranking/tests/test_feature_processing.py

```diff
@@ -0,0 +1,31 @@
+import string
+
+from ml4ir.applications.ranking.tests.test_base import RankingTestBase
+from ml4ir.base.features import preprocessing
+
+
+class RankingModelTest(RankingTestBase):
+    def test_text_preprocesing(self):
+        """
+        Unit test text preprocessing
```

Comment:
Can we make explicit in comments that here we test:
- punctuation removal
- lower casing

Reply:
Don't forget this.